app/controllers/assignments_controller.rb

```diff
@@ -1,27 +1,27 @@
 class AssignmentsController < ApplicationController
-	before_filter :userLoggedIn
-	before_filter :courseExists, :except => [:show]
-	before_filter :assignmentExists, :only => [:show, :edit, :update, :delete]
+  before_filter :userLoggedIn
+  before_filter :courseExists, :except => [:show]
+  before_filter :assignmentExists, :only => [:show, :edit, :update, :delete]
   before_filter :partOfCourse, :only => [:show]
-	before_filter :authCourseEdit, :except => [:show]
+  before_filter :authCourseEdit, :except => [:show]
 
   def show
-		@assignment = Assignment.find_by(id: params[:assignment_id])
+    @assignment = Assignment.find_by(id: params[:assignment_id])
   end
 
-	def new
-		#render the page to create new assignment
+  def new
+    # render the page to create new assignment
     @course = Course.find(params[:course_id])
-	end
+  end
 
   def create
     @assignment = Assignment.new(assignment_params)
 
     if @assignment.due_date < Date.today
       flash[:message] = "Cannot create an assignment that has already ended."
-      render 'new' and return  
+      render 'new' and return
```

Comment:
Use `&&` instead of `and`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.